internal/feature/flags.go

```diff
@@ -0,0 +1,7 @@
+package feature
+
+// Flags contains the values of the feature flags.
+type Flags struct {
+	// EnableUpgradeAgent is a boolean flag to enable or disable the upgrade agent feature:
+	EnableUpgradeAgent bool `envconfig:"ENABLE_UPGRADE_AGENT" default:"true"`
```

Comment:
/hold
I think we should start with default `false`
Otherwise, the service might send upgrade step to old agents that don't know what this step means...

Reply:
Done, changed to `false`.